mmv1/third_party/terraform/services/bigtable/resource_bigtable_table.go

```diff
@@ -103,6 +103,29 @@ func ResourceBigtableTable() *schema.Resource {
 				ValidateFunc: verify.ValidateDuration(),
 				Description:  `Duration to retain change stream data for the table. Set to 0 to disable. Must be between 1 and 7 days.`,
 			},
+
+			"automated_backup_policy": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"retention_period": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: verify.ValidateDuration(),
+							Description:  `How long the automated backups should be retained. The only supported value at this time is 3 days.`,
```

Comment:
That's a server side validation, right? We should remember to edit this text when we change the server side validation

Reply:
Thanks. That's right. To simplify future updates, I've removed the reference to particular values here